packages/dds/map/src/test/map.spec.ts

```diff
@@ -313,6 +326,38 @@ describe("Map", () => {
                 assert.equal(map1.get(key), newValue, "The first map did not get the new value");
                 assert.equal(map2.get(key), newValue, "The second map did not get the new value");
             });
+
+            it("metadata op", async () => {
+                // type IMapKeyOperation = IMapSetOperation | IMapDeleteOperation;
```

Comment:
I'm not sure what this comment and the one on line 359 are for.

Reply:
previous code iteration tracing. deleting...